src/main/java/frc/robot/commands/scoring/Stow.java

```diff
@@ -1,22 +1,22 @@
 package frc.robot.commands.scoring;
 
-import com.revrobotics.CANSparkMax.IdleMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
 
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-import frc.robot.constants.ArmConstants;
-import frc.robot.constants.ElevatorConstants;
 import frc.robot.commands.scoring.arm.ExtendArm;
 import frc.robot.commands.scoring.elevator.MoveElevator;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.ElevatorConstants;
 import frc.robot.subsystems.Elevator;
 import frc.robot.subsystems.FourBarArm;
-import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.RollerIntake;
 
 public class Stow extends SequentialCommandGroup {
-  public Stow(Intake intake, Elevator elevator, FourBarArm arm) {
+  public Stow(RollerIntake intake, Elevator elevator, FourBarArm arm) {
     addRequirements(intake, elevator, arm);
     addCommands(
-      new InstantCommand(() -> intake.setIdleMode(IdleMode.kBrake)),
+      new InstantCommand(() -> intake.setNeutralMode(NeutralMode.Brake)),
       new ExtendArm(arm, ArmConstants.kStowPos),
       new MoveElevator(elevator, ElevatorConstants.kStowHeight)
```

Comment:
I don't think this is necessary anymore, we don't need to set the intake to brake mode anymore so instead it can move as part of PositionRollerIntake